src/store/model/fnac.ts

```diff
@@ -0,0 +1,38 @@
+import {Store} from './store';
+
+export const Fnac: Store = {
+	labels: {
+		inStock: {
+			container: '.f-buyShopBox',
+			text: ['En stock en ligne']
+		},
+		maxPrice: {
+			container: 'span[class*="PriceString"]',
+			euroFormat: true
+		},
+		outOfStock: {
+			container: '.f-buyShopBox',
+			text: ['Indisponible en ligne']
+		}
+	},
+	links: [
+		{
+			brand: 'test:brand',
+			cartUrl:
+				'https://secure.nl.fnac.be/basket/add?from=111&productId=14119956',
+			model: 'test:model',
+			series: 'test:series',
+			url:
+				'https://www.fnac.com/Console-Sony-PS5-Edition-Standard/a14119956/w-4'
```

Comment:
Could you change to something that is a better test candidate? Something that is usually in stock?